config/database.yml

```diff
@@ -1,8 +1,14 @@
+# If you want to change this file, please keep the changes in your working
+# copy by using
+#
+#     git update-index --skip-worktree config/database.yml
```

Comment:
Why do you prefer env to --skip-worktree? This is exactly the use case for this flag, as you can read in the linked page above: Have developers change a fil to suit their local needs, but keep the original file forever in the repository. The skipped file will never be comitted, but if a remote change creates a conflit, then the user is asked to resolve the conflict locally, and the new version is still skipped. It's a win-win.

And, if you know that rails loves the DATABASE_URL env variable, you are done and you don't need any more instructions.

About having it in the wiki, I disagree for two reasons:

1. people don't read
2. people might fork the repo, and not the wiki
3. this is really specific to the database.yml file, and people is used to look in there to configure their local settings
4. people can't count.